index.html

```diff
@@ -1,200 +1,259 @@
+                        <p> JavaScript </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Ruby </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Html </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Css </p>
+                    </li>
+                </ul>
+            </li>
+            <li>
+                <div class="container categories">
+                    <img src="icons/square.svg" alt="">
+                    <h3> Frameworks </h3>
+                </div>
+                <ul class="container content knowledge">
+                    <li class="textPlaceholder element">
+                        <p> Bootstrap </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Ruby on Rails </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> RSPec </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> CapyBara </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Selenium </p>
+                    </li>
+                </ul>
+            </li>
+            <li>
+                <div class="container categories">
+                    <img src="icons/circle.svg" alt="">
+                    <h3> Skills </h3>
+                </div>
+                <ul class="container content knowledge">
+                    <li class="textPlaceholder element">
+                        <p> Codekit </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Github </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Codepen </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Gitlab </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Terminal </p>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+    </section>
+    <section class="container">
+        <div class="content contact">
+            <p> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+            <form class="contact" action="https://formspree.io/f/xqknzgnq" method="post">
+                <input name="user-name" type="text" id="name" placeholder="Full name" maxlength="30" required>
+                <input name="user-email" type="email" id="email" placeholder="Email address" required>
+                <textarea name="comment" maxlength="500">Enter text here</textarea>
```

Comment:
- [ ] Great job setting up formspree correctly 👏  you should consider adding "required" to the `<textarea>`, I was able to submit the form with a blank field.